docs/content_management/images/fastly_io.md

```diff
@@ -4,33 +4,17 @@ description: Configure Fastly Image Optimizer.
 
 # Fastly Image Optimizer (Fastly IO)
 
-The Fastly Image Optimizer (Fastly IO) is an external service that provides real-time image optimisation for multiple input and output formats.
+The Fastly Image Optimizer (Fastly IO) is an external service that provides real-time image optimization for multiple input and output formats.
 It serves and caches image requests from your origin server, making your website faster and more efficient.
 
 To be able to configure this feature, you need [Fastly IO subscription](https://docs.fastly.com/en/guides/about-fastly-image-optimizer).
 
 ## Enable shielding
 
-To use Fastly Image Optimizer, you need shielding.
-To enable it, follow steps in Fastly documentation, [Enabling and disabling shielding](https://developer.fastly.com/learning/concepts/shielding/).
-
-Before proceeding, make sure that you have the `snippet_re_enable_shielding.vcl` configuration file added to your project:
-
-```vcl
-set var.fastly_req_do_shield = (req.restarts <= 2);
-
-# set var.fastly_req_do_shield = (req.restarts > 0 && req.http.accept == "application/vnd.fos.user-context-hash");
-set req.http.X-Snippet-Loaded = "v1";
-```
-
-You can add it using Fastly CLI: 
- 
-```bash
-- fastly vcl snippet create --name="Re-Enable shielding on restart" --version=active --autoclone --priority 100 --type recv --content=vendor/ibexa/fastly/fastly/snippet_re_enable_shielding.vcl
-- fastly service-version activate --version=latest
-```
-
-Next, you need to choose the [Shield location](https://developer.fastly.com/learning/concepts/shielding/#choosing-a-shield-location) from the Shielding menu in Fastly web interface as specified in [Fastly IO documentation](https://docs.fastly.com/en/guides/shielding#enabling-shielding).
+To use Fastly Image Optimizer, you first need a [working setup of Ibexa DXP and Fastly](reverse_proxy.md#using-varnish-or-fastly)
+with shielding enabled.
+To enable shielding, follow the steps in [Fastly developer documentation](https://developer.fastly.com/learning/concepts/shielding/#enabling-and-disabling-shielding).
+Remember to choose a shield location from the **Shielding** menu, as described in [Fastly user documentation](https://docs.fastly.com/en/guides/shielding#enabling-shielding).
```

Comment:
```suggestion
Remember to choose a shield location from the **Shielding** menu, as described in [Fastly documentation](https://docs.fastly.com/en/guides/shielding#enabling-shielding).
```
To differentiate from our user documentation

Reply:
I actually introduced "user" here, because one sentence up we are linking to Fastly developer doc.